### PR TITLE
corrected Issue #18297

### PR DESCRIPTION
**Pull request by jzwickel**

### CODE REVIEW

File: tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md [hunk 1]
```diff
@@ -190,31 +190,27 @@ After you are done, deploy the whole project again. When that is completed succe
...
+2.	Paste the following statement in the SQL Console. Change the password in the statement and then run.
 
     ```SQL
 CREATE USER report PASSWORD <your_password> NO FORCE_FIRST_PASSWORD_CHANGE set usergroup default;
 ```
 
     > Using the clause `NO FORCE_FIRST_PASSWORD_CHANGE` is not considered a security best practice! We will only use this option for the purpose of this tutorial, in our example to make a user available to multiple individuals. If you create users in your productive environment, please consider forcing a password change for the first log in and giving individuals different users.
 
-4.	Now that our new user `report` is created, we need to grant the user access to the role `PublicAccessSchema`.
+3.	Now that our new user `report` is created, we need to grant the user access to the role `PublicAccessSchema`.
```
Comment (MichaelCzcz) [inline tutorials/hana-cloud-mission-trial-9/hana-cloud-mission-trial-9.md:203]: Please don't say "we" and "our" unless you are referring to SAP.
Please change all occurrences

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for your pull request. Please address the comment in line 203 and update the tutorial tags. The name space "products" is no longer valid. Please use "software-product" instead.

**Comment by MichaelCzcz** [comment]
@jzwickel Please address the requested changes, otherwise I cannot merge this pull request.

**Comment by jzwickel** [comment]
@MichaelCzcz your requested changes were independent of the reported issue. So in fact it would have been cleaner to deal with them separately. 
As you know, following the process with Atom is pretty cumbersome and takes considerably time. I will push the changes when I have solved the process issue.
Best,
Jan

**Pull request merged**